oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elasticsearch/ElasticsearchIndexProviderService.java

```diff
@@ -140,6 +149,8 @@ private void activate(BundleContext bundleContext, Map<String, ?> config) {
         //initializeExtractedTextCache(config, statisticsProvider);
 
         elasticsearchConnection = getElasticsearchCoordinate(config);
+        indexPrefix = PropertiesUtil.toString(config.get(PROP_INDEX_PREFIX), PROP_INDEX_PREFIX_DEFAULT);
+        LOG.info("Got indexPrefix=" + indexPrefix);
```

Comment:
Can we change this to something like LOG.info("IndexPrefix - {} configured for Elastic Search indices", indexPrefix)

Reply:
I actually deleted it as I didn't intend to add it. May be I can re add it. what about level - Info or debug?

Reply:
debug should be ok I suppose